UiPath.FreeRdpClient/UiPath.FreeRdpClient.Tests/TestInfra/TestHost.cs

```diff
@@ -26,6 +27,8 @@ public class TestHost : IServiceProvider, IHost, IAsyncDisposable
 
     private IHost _host => _hostLazy.Value;
 
+    public Wts Wts { get; } = new();
```

Comment:
I would keep the testhost as an all purpose tool and make this Wts an extension to be used in freerdp tests